File: Feb_6th,7th_Tasks/Python programs/StudentMarks.py
marks = [85, 110, 75, -5, 40, 60]

invalid = 0
pass_count = 0
fail_count = 0

for m in marks:
    if m < 0 or m > 100:
        invalid += 1
    elif m >= 50:
        pass_count += 1
    else:
        fail_count += 1

print("Invalid Marks:", invalid)
print("Passed:", pass_count)
print("Failed:", fail_count)

File: Feb_6th,7th_Tasks/Java programs/StudentMarks.java
public class StudentMarks {
    public static void main(String[] args) {
        int[] marks = {85, 110, 75, -5, 40, 60};

        int invalid = 0, pass = 0, fail = 0;

        for (int m : marks) {
            if (m < 0 || m > 100) {
                invalid++;
            } else if (m >= 50) {
                pass++;
            } else {
                fail++;
            }
        }

        System.out.println("Invalid Marks: " + invalid);
        System.out.println("Passed Students: " + pass);
        System.out.println("Failed Students: " + fail);
    }
}
